MetaMask/metamask-extension
# `TextField` house keeping 🧹

## Explanation

Updating `TextField` to include the most up to date standards and conventions for the component-library components listed in the issue and description below

* Fixes #16642  

- [ ] Add locals for any default text I18nContext as default context **Leaving this for now it need help with localization I'll create another issue**

## Screenshots/Screencaps

### Before

https://user-images.githubusercontent.com/8112138/203663513-a158d920-4875-433c-9ec8-52775a801d4e.mov

### After

https://user-images.githubusercontent.com/8112138/203663541-9f2088ec-16d2-4e17-b2b2-80907a3aaa48.mov

## Manual Testing Steps

- Go to latest build of Storybook on this PR
- Search `TextField` in the search bar
- Check stories, controls and docs

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
